samples/matter/light_switch/sample.yaml

```diff
@@ -5,7 +5,6 @@ tests:
   # Excluded in quarantine.yaml to limit resources usage in integration builds
   sample.matter.light_switch.debug:
     build_only: true
-    extra_args: CONFIG_CHIP_LIB_SHELL=y
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
```

Comment:
Maybe add nrf7002dk_nrf5340_cpuapp to integration platforms for consistency.

Reply:
There is a separate `sample.matter.light_switch.debug.nrf7002dk` test for that, right? I thought that was the convention.

Reply:
I will merge this PR now, as it took some time to get a green CI, but I can change it in a separate PR.